lib/argsert.ts

```diff
@@ -73,7 +73,7 @@ export function argsert(
       position += 1;
     });
   } catch (err) {
-    console.warn(err.stack);
+    console.warn((err as Error).stack);
```

Comment:
👋 hi all, here my 50cent, sorry!

The scope of usage `unknown` instead of `any` is to prevent fatal errors when catch receive something what is not an `Error`.

`"errorString".stack` > fatal error
to avoid this i suggest to add a check if `err` is instance of `Error` like suggested by documentation instead of cast `err as Error`

https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables